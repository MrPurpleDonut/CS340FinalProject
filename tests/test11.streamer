text contents << args[0];
replaceAll(contents, "teh", "the");
contents >> args[0];
