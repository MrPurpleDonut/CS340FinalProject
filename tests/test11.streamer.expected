Program parsed correctly!
---------------------------
--------Printing AST-------
---------------------------
Program
    Function List
    Statement List
        Declaration statement: contents
            Type: text
            File Read into variable: contents
                List Access
                    Identifier: args
                    Number: 0.0
        Statement as Expression
            Function call: replaceAll
                Expression List
                    Identifier: contents
                    Quoted string: "teh"
                    Quoted string: "the"
        File Write Statement
            Identifier: contents
            List Access
                Identifier: args
                Number: 0.0
